cmd/up.go

```diff
@@ -158,41 +183,141 @@ func (cmd *UpCmd) Run(ctx context.Context, devPodConfig *config.Config, client c
 		log.Infof("Run 'ssh %s.devpod' to ssh into the devcontainer", client.Workspace())
 	}
 
+	dotfilesRepo := devPodConfig.ContextOption(config.ContextOptionDotfilesURL)
+	if cmd.DotfilesSource != "" {
+		dotfilesRepo = cmd.DotfilesSource
+	}
+
+	dotfilesScript := devPodConfig.ContextOption(config.ContextOptionDotfilesScript)
+	if cmd.DotfilesScript != "" {
+		dotfilesScript = cmd.DotfilesScript
+	}
+
+	if dotfilesRepo != "" {
+		log.Infof("Dotfiles repo %s specified", dotfilesRepo)
+		log.Infof("Setting dotfiles into the devcontainer")
+
+		execPath, err := os.Executable()
+		if err != nil {
+			return err
+		}
+
+		agentArguments := []string{
+			"--debug",
```

Comment:
Only use debug if cmd.Debug is true please

Reply:
Done :+1: